src/EventStore.Core.Tests/ClientAPI/subscribe_to_all_filtered_should.cs

```diff
@@ -222,12 +226,12 @@ public class subscribe_to_all_filtered_should<TLogFormat, TStreamId> : Specifica
 						}
 						return Task.CompletedTask;
 					},
-					2)) {
+					checkpointInterval: 2)) {
 					await _conn.AppendToStreamAsync("stream-a", ExpectedVersion.NoStream, _testEvents);
 
 					await appeared.Task.WithTimeout(Timeout);
 
-					Assert.AreEqual(5 /*checkpoints*/ * 2 /*considered events*/ - (isV3 ? 1 : 0) /*filtered out "stream-a" stream record event*/, eventsSeen.Count);
+					Assert.AreEqual(5 /*checkpoints*/ * 2 /*considered events*/, eventsSeen.Count);
```

Comment:
is it known why this was flaky the was it was before?

Reply:
Same as other test, events might still be counted after reaching the checkpoint count as there are more events present.